src/palette/qml/MuseScore/Palette/internal/AddPalettesPopup.qml

```diff
@@ -89,18 +89,20 @@ StyledPopupView {
         StyledListView {
             id: palettesList
             height: Math.min(availableHeight, contentHeight)
-            width: parent.width
+            width: parent.width + margins
```

Comment:
This should be root.margins. You should always specify the object to which the property belongs. It's better for readability/performance reasons (and less chance of making a mistake)